packages/block-editor/src/components/link-control/search-input.js

```diff
@@ -81,7 +81,6 @@ const LinkControlSearchInput = forwardRef(
 				...props,
 				instanceId,
 				withCreateSuggestion,
-				currentInputValue: value,
```

Comment:
Sounds like something that needs to be mentioned in the CHANGELOG?

Reply:
Yes, I now added info to `block-editor` about the `URLInput.renderSuggestions` having a new `currentInputValue` parameter.